src/components/Ayah/index.js

```diff
@@ -37,6 +37,95 @@ export default class Ayah extends Component {
     isSearched: false
   };
 
+  componentDidMount() {
+
+    function getOffset(elem) {
```

Comment:
I'd recommend putting this function as a util. We should aim to keep component file simple as possible.